playwright-e2e/tests/smoke-test.spec.ts

```diff
@@ -69,6 +71,12 @@ test("Smoke Test @smoke-test", async ({
   await addApplicationDocuments.uploadDocumentSmokeTest();
   await startApplication.addApplicationDocumentsInProgress();
 
+  await page.pause();
+  // Add respondents' details
+  await startApplication.respondentsDetailsLink.isVisible();
+  await startApplication.respondentDetails();
+  await respondentDetails.respondentDetailsNeeded();
```

Comment:
May be 'await startApplication.respondentsDetailsLink.isVisible();' can be added into startApplication.respondentDetails() method?

Reply:
i think this was actually duplicated perhaps so have removed.